Khan/wonder-blocks
# Add a11y features to banners

## Summary:
- Add role values to banner
- Add aria-live values to banner
- Add an aria label for the dismiss button
- Add an aria label to the icons
- Add tests for accessibility

**Storybook publish:**
https://5e1bf4b385e3fb0020b7073c-jnsslrwmuu.chromatic.com/?path=/docs/banner--default

**Chromatic:**
https://www.chromatic.com/build?appId=5e1bf4b385e3fb0020b7073c&number=2396

Issue: https://khanacademy.atlassian.net/browse/WB-1385

### Demo: Icon and button aria labels
https://user-images.githubusercontent.com/13231763/197654765-8a0b8ed4-d94c-41f4-b5b5-06db9dba13ed.mov

### Demo: Different icon labels for different icons
https://user-images.githubusercontent.com/13231763/197654799-5723ddb2-75a6-45c6-be75-937b8908dfcc.mov

### Demo: assertive role makes the screen reader read out banner on render
https://user-images.githubusercontent.com/13231763/197654885-cc3b5c20-9d22-4cbc-83b3-97fca6054673.mov


## Test plan:
- yarn jest `packages/wonder-blocks-banner/src/components/__tests__/banner.test.js`

Storybook
- Go to Storybook at http://localhost:6061/?path=/story/banner--default
- Go through all the stories and use a screenreader with all of them
- Confirm that the icons and buttons have accessible labels
- Use the rotor to search by role. Confirm that banners show up under their expected roles.
